sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/AccessTier.java

```diff
@@ -52,6 +52,9 @@ public final class AccessTier extends ExpandableStringEnum<AccessTier> {
     /** Static value Archive for AccessTier. */
     public static final AccessTier ARCHIVE = fromString("Archive");
 
+    /** Static value Premium for AccessTier. */
+    public static final AccessTier PREMIUM = fromString("Premium");
+
```

Comment:
This looks like a gap . .NET has it 
https://github.com/Azure/azure-sdk-for-net/blob/5d32fe9104d352ffe50e0e9e62b3de09d5484995/sdk/storage/Azure.Storage.Blobs/src/Generated/Models/AccessTier.cs#L70